legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/result/Result.java

```diff
@@ -37,6 +39,11 @@ public Result(String status, List<ExecutionActivity> activities)
         this.activities = activities;
     }
 
+    public void addChildrenResults(MutableList<Result> childResults)
```

Comment:
Shouldn't close method close these results?

Also, why 'children' in method name and 'child' in attribute name?